workflow-templates/mendix-actions.yml

```diff
@@ -0,0 +1,43 @@
+## Ensure that all lines with comments are filled out correctly ##
+
+name: Actions
+
+on:
+  push:
+    branches:
+      - "development"
+      - "test"
+      - "main"
+  pull_request:
+    branches:
+      - "development"
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    name: Mendix
+    uses: vermeer-corp/it-mendix-gitrepotesting/.github/workflows/mendix-build-and-deploy-on-cloud.yml@development ## Confirm repo name i.e. it-mendix-appname
```

Comment:
Also, should this use the [`$default-branch` ](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow) token instead of hardcoding `development`?

Reply:
I think development should be used, but default branch works as dev should be set as default anyways.